app/src/main/java/com/sun/music_66/MusicApplication.kt

```diff
@@ -0,0 +1,21 @@
+package com.sun.music_66
+
+import android.app.Application
+import android.content.Context
+
+class MusicApplication : Application() {
+
+    companion object {
+        lateinit var instance: MusicApplication
+            private set
+        val context: Context
```

Comment:
khi sử dụng lateinit bắt buộc phải check xem biến đó đã được khởi tạo hay chưa
```suggestion
        val context: Contextval applicationContext: Context?
            get() = if (::INSTANCE.isInitialized) INSTANCE.applicationContext else null
```